<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camshift_Multiple_Trackers</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>Webcam</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="infoLabel">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Browse for a video file or use the video feed from your webcam.&lt;/p&gt;&lt;p&gt;Press &amp;quot;p&amp;quot; to pause or unpause the video.&lt;/p&gt;&lt;p&gt;Press &amp;quot;k&amp;quot; to destroy(clean up) all trackers.&lt;/p&gt;&lt;p&gt;Press &amp;quot;ESC&amp;quot; to stop the video.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>383</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="actionDestroy_All_Windows"/>
    <addaction name="actionDestroy_All_Trackers"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Me"/>
   </widget>
   <addaction name="menuWindows"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionDestroy_All_Windows">
   <property name="text">
    <string>Destroy All Windows</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Backspace</string>
   </property>
  </action>
  <action name="actionDestroy_All_Trackers">
   <property name="text">
    <string>Destroy All Trackers</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="actionAbout_Me">
   <property name="text">
    <string>About Me</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
